### PR TITLE
split export function into two separate functions

### PR DESCRIPTION
**Pull request by Atyanar**
export_overall_course_rating delivers general information
export_course_evaluations delivers paginated course evaluations

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -46,3 +46,7 @@ rubocop.html
 
 # Ignore auto-generated assets
 /public/assets
+
+# Ignore passenger files
```
Comment (MrSerth) [inline .gitignore]: Could you please ignore these files in a more general way?
For example, you could either all passenger.*.pid files or all *.pid files here.

**Comment by MrSerth** [comment]
I am fine with your changes. Except for the failing specs, everything looks good to me. Please review the three failing specs and try to fix them (preferred to remove them).

**Comment by Dome-GER** [comment]
Looks good to me :+1:

File: .gitignore [hunk 2]
```diff
@@ -46,3 +46,6 @@ rubocop.html
...
+passenger.*
```
Comment (MrSerth) [inline .gitignore]: Thanks for your edit! What about a new line character here?

**Pull request merged**